src/Microsoft.AspNetCore.WebHooks.Receivers.Dropbox/Filters/DropboxVerifySignatureFilter.cs

```diff
@@ -78,10 +78,17 @@ public class DropboxVerifySignatureFilter : WebHookVerifySignatureFilter, IAsync
                     return;
                 }
 
-                var expectedHash = GetDecodedHash(header, DropboxConstants.SignatureHeaderName, out errorResult);
-                if (errorResult != null)
+                byte[] expectedHash;
+                try
                 {
-                    context.Result = errorResult;
+                    expectedHash = FromHex(header);
+                }
+                catch (Exception exception)
```

Comment:
Is it possible to catch formatexception or something more specific here? I'm having flashbacks of the dreaded catch-in-input-formatters